Gemfile

```diff
@@ -19,43 +19,16 @@ gem 'mysql2', '~> 0.3.2'
 
 # Use SCSS for stylesheets
 gem 'sass-rails', '~> 5.0'
-gem 'bootstrap-sass', '~> 3.3.4.1'
```

Comment:
Cannot see this being used anywhere. Seems like we have our own customized css (sufia might be using it but that's their own dependencies)